### DISCUSSION TITLE
When is `_count` of relation count null ?

### DISCUSSION THREAD

**Discussion by mitsuru-kasai-exwzd** [prisma/prisma#9374] [Q&A] [answered]
Hi there, 

I am so grateful for release of Prisma 3.0 and GA of relation count!

Then, I have a question about relation count.

Using relation count query, for example
```ts
const modelA = await prisma.modelA.findMany({
    include: {
        _count: { select: { modelB: true } },
    },
});
```
, `_count` field is nullable.

Why is `_count` field nullable?
Or, when is `_count` filed null?

I'm sorry for my poor english, and sorry if the question is ambiguous.

Thanks in advance!

**Comment by ryands17** [comment, marked as answer]
@mitsuru-kasai-exwzd 👋 

`_count` will never be `null` so this is a bug. It would be great if you could create one [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that we can look into this 🙂

**Reply by mitsuru-kasai-exwzd** [reply]
I see!

I will create a issue, thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
